site/layouts/partials/incident.html

```diff
@@ -6,9 +6,10 @@
   <div class="incident-description">
     {{- .Content -}}
   </div>
-  {{- $alert := index .Site.Data.severity.alerts .Params.severity -}}
+  {{- $severity := .Params.severity  | default "under-maintenance" -}}
```

Comment:
I wonder if this is the most appropriate default choice. What was your reasoning in choosing this one?

Reply:
I chose this one because it is the least severe item that didn't say everything was ok. Since the default _should_ never be used, I didn't want to create a new description or alert and figured it would be ok.

Reply:
Make sense to me. Thanks!